app/views/catalog/_index_header_list_collection.html.erb

```diff
@@ -1,3 +1,4 @@
-  <h2>
-    <%= link_to document.title_or_label, [hyrax, document] %>
-  </h2>
+<div class="search-results-title-row">
+  <h3 class="search-result-title"><%= link_to document.title_or_label, [hyrax, document] %></h3>
+  <span class="label label-success">User Collection (wire this up)</span>
```

Comment:
Where does this show up in the UI?  It looks like `User Collection (wire this up)` either needs to be removed or something needs to be done here.